main.rb

```diff
@@ -1,37 +1,30 @@
 require_relative 'app'
```

Comment:
![image](https://github.com/DianaBeki/school-library/assets/93039359/49f5fed6-f3f6-4926-b1b7-f22b3dd89055)

- [ ] Great work so far but it would be nice if you make a method to return the inputs and you can use that method everywhere instead of writing gets.chomp and puts again and again because it is against the DRY principal 😨 Kindly make a method which should handle this case 👍 You can use the below method for this purpose ⬇️ 👀


```ruby
module UserInput
  def user_input(messages)
    inputs = []
    messages.each do |msg|
      print "#{msg}: "
      inputs.push(gets.chomp)
    end
    inputs
  end
end
```